README.md

```diff
@@ -65,8 +65,6 @@ After you're finished please remove all the comments and instructions!
 > In this project, I will set up a "Hello world" repository and message. My goal here is to master all of the tools and best practices that I learned in Microverse. 
 I hope It demostrates my profesionalism and my capability to document any project in a rigth way!
 
```

Comment:
:orange_circle: [OPTIONAL] Kindly make sure you remove the descriptions placeholders :arrow_down: They are just there to tell you what will be the content of the section.
- ![Screenshot from 2023-01-18 07-01-14](https://user-images.githubusercontent.com/26736582/213088523-186448b0-9c27-4b45-9652-0e393667a1ba.png)
- ![Screenshot from 2023-01-18 07-01-02](https://user-images.githubusercontent.com/26736582/213088525-a6f0cdc0-0f1f-48dd-b6a4-e6836815cd6c.png)
- ![Screenshot from 2023-01-18 07-00-51](https://user-images.githubusercontent.com/26736582/213088529-82c0bec8-1bc1-48e6-b38a-2f7ba4e45016.png)
- ![Screenshot from 2023-01-18 07-00-40](https://user-images.githubusercontent.com/26736582/213088533-4648404e-c68a-4840-9f7f-31d6f775285f.png)
- ![Screenshot from 2023-01-18 07-00-24](https://user-images.githubusercontent.com/26736582/213088535-3a15e34d-cff9-4625-a3d7-a96c87391f4a.png)
- ![Screenshot from 2023-01-18 07-00-10](https://user-images.githubusercontent.com/26736582/213088537-5abf8d52-1b97-47fc-9c64-7cd9e8c4ce02.png)